### PR TITLE
Update CI fields and add badge to README

### LINKED ISSUE #206: Migrate CI/CD Github actions to helium-csharp
## Description:
- Move DocerCI.yml and DockerCIByPR.yml to retaildevcrews/helium-csharp

### PR DESCRIPTION
**Pull request by evillgenius75**
## Type of PR

- [ ] Documentation changes
- [ ] Code changes
- [ ] Test changes
- [ X] CI-CD changes
- [ ] GitHub Template changes

## Purpose of PR
Update CI to Helium Standard process. Add badge to Readme for CI Validation
## Issues Closed or Referenced

- Closes retaildevcrews/helium#206
- References retaildevcrews/helium#200

### CODE REVIEW

File: .github/workflows/dockerByPR.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
+name: DockerCI-PR
+
+on:
+  pull_request:
+    branches:
+      - master
+
+    paths-ignore:
+      - 'docs/**'
+      - '.github/**'
```
Comment (aflinchb) [inline .github/workflows/dockerByPR.yml:10]: Didn't notice this in the other reviews (for java and typescript) but will probably have to go back and update this list. 

This should include I think all of the md files, can you do *.md?

**Review by aflinchb — APPROVED** [review]
Approved with comments on changing paths-ignore in yml files.

File: .github/workflows/dockerByPR.yml (hunk 1 above)
Comment (evillgenius75) [reply .github/workflows/dockerByPR.yml:10]: Fixed with commit 5f40b20

**Review by dsturgell — APPROVED** [review]

**Review by wereinse — APPROVED** [review]
LGTM

**Pull request merged**